bin/docker-image-tool.sh

```diff
@@ -102,33 +114,37 @@ function build {
     error "Failed to build Spark JVM Docker image, please refer to Docker build output for details."
   fi
 
-  docker build $NOCACHEARG "${BINDING_BUILD_ARGS[@]}" \
-    -t $(image_ref spark-py) \
-    -f "$PYDOCKERFILE" .
-    if [ $? -ne 0 ]; then
-      error "Failed to build PySpark Docker image, please refer to Docker build output for details."
+  if [ "${PYDOCKERFILE}" != "skip" ]; then
+    docker build $NOCACHEARG "${BINDING_BUILD_ARGS[@]}" \
+      -t $(image_ref spark-py) \
+      -f "$PYDOCKERFILE" .
+      if [ $? -ne 0 ]; then
+        error "Failed to build PySpark Docker image, please refer to Docker build output for details."
+      fi
+  else
+    echo "Skipped building PySpark docker image."
+  fi
+
+  if [ "${RDOCKERFILE}" != "skip" ]; then
+    if [ -d "${SPARK_HOME}/R/lib" ]; then
+      docker build $NOCACHEARG "${BINDING_BUILD_ARGS[@]}" \
+        -t $(image_ref spark-r) \
+        -f "$RDOCKERFILE" .
+      if [ $? -ne 0 ]; then
+        error "Failed to build SparkR Docker image, please refer to Docker build output for details."
```

Comment:
Again, script should bail out i.e. `exit` if the image build fails

Reply:
Same comment as above. Script does exit with non-zero exit code in this case.